src/imgix-core-js.js

```diff
@@ -70,6 +70,72 @@
       return this.settings.urlPrefix + this.settings.domain + path + queryParams;
     };
 
+    ImgixClient.prototype.buildSrcSet = function (path, params) {
+      var width = params['w'];
+      var height = params['h'];
+      var aspectRatio = params['ar'];
+      var aspectRatioAsDecimal = function() {
+        if (typeof aspectRatio !== "string") {
+          return false;
+        }
+        var isValidFormat = function(str) {
+          return /^\d+(\.\d+)?:\d+(\.\d+)?$/.test(str);
+        };
+        if (!isValidFormat(aspectRatio)) {
+          return false;
+        }
+      
+        const aspectRatioSplit = aspectRatio.split(":");
+        var arWidth = aspectRatioSplit[0];
+        var arHeight = aspectRatioSplit[1];
+      
+        return parseFloat(arWidth) / parseFloat(arHeight);
+      }();
+
+      var fixedWidth = ((width && height) || (width && aspectRatio) || (height && aspectRatio)) ? true : false;
+
+      if (fixedWidth) {
+        if (width && height) {
+          return this._buildDPRSrcSet(path, params);
+        }
+        else if (aspectRatio) { 
```

Comment:
At first I thought that these aspect ratio calculations were necessary but the more I mess around with the `ar` parameter, the less I think this is actually needed.